src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3356,6 +3356,11 @@ namespace ZoneTempPredictorCorrector {
 				LoadToCoolingSetPoint = AIRRAT( ZoneNum ) * ( TempZoneThermostatSetPoint( ZoneNum ) - ZoneT1( ZoneNum ) ) + TempDepZnLd( ZoneNum ) * TempZoneThermostatSetPoint( ZoneNum ) - TempIndZnLd( ZoneNum );
 			}
 			if ( RAFNFrac > 0.0 ) LoadToHeatingSetPoint = LoadToHeatingSetPoint / RAFNFrac;
+
+			//
+			if ( DataHeatBalance::Zone( ZoneNum ).HasApproachTempToReturnAir && !( DataGlobals::BeginSimFlag ) ) {
+				LoadToCoolingSetPoint = TempDepZnLd( ZoneNum ) * DataHeatBalance::Zone( ZoneNum ).AdjustedTempToReturnAir - TempIndZnLd( ZoneNum );
+			}
```

Comment:
This line (and similar ones below) are new?  If these lines are correct, then zones with HasApproachTempToReturnAir can't have *any* other HVAC equipment besides a single VAV terminal unit and the whole idea of what the load is changes.  Not sure about this.